### PR TITLE
feat(lite-chains): include definitions

### PR DESCRIPTION
**Pull request by james-a-morris**
This UMIP change adds references to Across Lite Chains.

### CODE REVIEW

File: UMIPs/umip-179.md [hunk 1]
```diff
@@ -33,6 +33,11 @@ The following sections from UMIP-157 are explicitly retained for use in Across v
 - [Proposal Information](https://github.com/UMAprotocol/UMIPs/blob/7b1a046098d3e2583abd0372c5e9c6003b46ad92/UMIPs/umip-157.md#proposal-information)
 - [Determining the Result](https://github.com/UMAprotocol/UMIPs/blob/7b1a046098d3e2583abd0372c5e9c6003b46ad92/UMIPs/umip-157.md#determining-the-result)
 
+## Global Constants
+All global constants from UMIP-157 will be retained for use in Across v3. The global variables stored in addition to UMIP-157 in the [AcrossConfigStore](https://etherscan.io/address/0x3b03509645713718b78951126e0a6de6f10043f5#code) are:
+* "LITE_CHAIN_ID_INDICES"
+    * This should be a stringified list of chain id numbers and these chains should be contained in the "CHAIN_ID_INDICES" list. There should be no duplicates in this list.
```
Comment (nicholaspai) [inline UMIPs/umip-179.md:39]:
```suggestion
    * This should be a stringified list of chain ID numbers and these chains should be contained in the "CHAIN_ID_INDICES" list. There should be no duplicates in this list and the list of chain IDs here should be a subset of CHAIN_ID_INDICES.
```
```suggestion
    * This should be a stringified list of chain id numbers and these chains should be contained in the "CHAIN_ID_INDICES" list. There should be no duplicates in this list. 
```

File: UMIPs/umip-179.md [hunk 2]
```diff
@@ -33,6 +33,11 @@ The following sections from UMIP-157 are explicitly retained for use in Across v
 - [Proposal Information](https://github.com/UMAprotocol/UMIPs/blob/7b1a046098d3e2583abd0372c5e9c6003b46ad92/UMIPs/umip-157.md#proposal-information)
 - [Determining the Result](https://github.com/UMAprotocol/UMIPs/blob/7b1a046098d3e2583abd0372c5e9c6003b46ad92/UMIPs/umip-157.md#determining-the-result)
 
+## Global Constants
+All global constants from UMIP-157 will be retained for use in Across v3. The global variables stored in addition to UMIP-157 in the [AcrossConfigStore](https://etherscan.io/address/0x3b03509645713718b78951126e0a6de6f10043f5#code) are:
+* "LITE_CHAIN_ID_INDICES"
+    * This should be a stringified list of chain ID numbers and these chains should be contained in the "CHAIN_ID_INDICES" list. There should be no duplicates in this list and the list of chain IDs here should be a subset of CHAIN_ID_INDICES.
```
Comment (nicholaspai) [inline UMIPs/umip-179.md:39]:
```suggestion
    * This should be a stringified list of chain ID numbers and these chains should be contained in the "CHAIN_ID_INDICES" list. There should be no duplicates in this list.
```

File: UMIPs/umip-179.md [hunk 3]
```diff
@@ -196,6 +201,9 @@ Note:
 ### Identifying SpokePool Contracts
 The current SpokePool address for a specific chain is available by querying `HubPool.crossChainContracts()`. The chainId must be specified in the query. In case of SpokePool migations, historical SpokePool addresses can be identified by scraping HubPool `CrossChainContractsSet` events.
 
+### Identifying a chains inclusion as a Lite Chain
+We consider a deposit to originate or be destined to a lite chain if the "LITE_CHAIN_ID_INDICES" constant in the AcrossConfigStore has an entry at a deposit's `quoteTimestamp` field.
```
Comment (nicholaspai) [inline UMIPs/umip-179.md:205]:
```suggestion
We consider a deposit to originate or be destined to a "Lite chain" if the "LITE_CHAIN_ID_INDICES" constant in the AcrossConfigStore has an entry at a deposit's `quoteTimestamp` field. These chains will have constraints for relayer repayments and slow fills to support faster development and expansion for the protocol.
```

File: UMIPs/umip-179.md [hunk 4]
```diff
@@ -196,6 +201,9 @@ Note:
 ### Identifying SpokePool Contracts
 The current SpokePool address for a specific chain is available by querying `HubPool.crossChainContracts()`. The chainId must be specified in the query. In case of SpokePool migations, historical SpokePool addresses can be identified by scraping HubPool `CrossChainContractsSet` events.
 
+### Identifying a chains inclusion as a Lite Chain
```
Comment (nicholaspai) [inline UMIPs/umip-179.md:204]:
```suggestion
### Identifying a chains inclusion as a "Lite" Chain
```

File: UMIPs/umip-179.md [hunk 5]
```diff
@@ -196,6 +201,9 @@ Note:
 ### Identifying SpokePool Contracts
 The current SpokePool address for a specific chain is available by querying `HubPool.crossChainContracts()`. The chainId must be specified in the query. In case of SpokePool migations, historical SpokePool addresses can be identified by scraping HubPool `CrossChainContractsSet` events.
 
+### Identifying a chains inclusion as a "Lite" Chain
+We consider a deposit to originate or be destined to a "Lite chain" if the "LITE_CHAIN_ID_INDICES" constant in the AcrossConfigStore has an entry at a deposit's `quoteTimestamp` field. These chains will have constraints for relayer repayments and slow fills to support faster development and expansion for the protocol.
```
Comment (nicholaspai) [inline UMIPs/umip-179.md:205]:
```suggestion
We consider a deposit to "originate" or be "destined to" a "Lite chain" if the "LITE_CHAIN_ID_INDICES" value in the AcrossConfigStore includes the deposit's origin chain or destination chain respectively as of the deposit's `quoteTimestamp` field. These chains will have constraints for relayer repayments and slow fills to support faster development and expansion for the protocol.
```

File: UMIPs/umip-179.md [hunk 6]
```diff
@@ -233,6 +241,8 @@ For the purpose of computing slow fills to be issued to recipients, each `Reques
 2. The `fillDeadline` has not already elapsed relative to the `destinationChainId` bundle end block number,
 3. The destination SpokePool `FillStatus` mapping for the relevant `V3RelayData` hash is `SlowFillRequested`,
 4. The `RequestedV3SlowFill` `V3RelayData` is matched by a corresponding `V3FundsDeposited` event on the origin SpokePool,
+5. Slow Fills requested for a deposit originating on a lite chain are considered invalid,
+6. Slow fills requested on a lite chain are considered invalid,
```
Comment (nicholaspai) [inline UMIPs/umip-179.md:245]:
```suggestion
6. The `destinationChainId` is not a Lite chain.
```

File: UMIPs/umip-179.md [hunk 7]
```diff
@@ -233,6 +241,8 @@ For the purpose of computing slow fills to be issued to recipients, each `Reques
 2. The `fillDeadline` has not already elapsed relative to the `destinationChainId` bundle end block number,
 3. The destination SpokePool `FillStatus` mapping for the relevant `V3RelayData` hash is `SlowFillRequested`,
 4. The `RequestedV3SlowFill` `V3RelayData` is matched by a corresponding `V3FundsDeposited` event on the origin SpokePool,
+5. Slow Fills requested for a deposit originating on a lite chain are considered invalid,
```
Comment (nicholaspai) [inline UMIPs/umip-179.md:244]:
```suggestion
5. The `originChainId` and `destinationChainId` are not Lite chains.
```

**Review by pxrl — COMMENTED** [review]
Looks good - a couple of minor comments.

**Review by amateima — APPROVED** [review]

**Review by mrice32 — APPROVED** [review]

**Pull request merged**